### ISSUE TITLE
Arm Architecture Build

### ISSUE THREAD

**Issue by c1rca** [NginxProxyManager/nginx-proxy-manager#2443]
It would be great if we could run nginx-proxy-manager:bullseye on arm builds. When I try and build it on oracle's ampere cloud it fails to exec bash.

**Comment by stijnos1991** [comment]
Yes this would be great. It seems to fail on ARM somehow. I can find additional related issues here on Github. I'm also using Ampere at Oracle.

**Comment by Coast47** [comment]
Can't run latest npm on arm architecture because it needs rust>=1.48 and latest buster package version is 1.41


`      error: Rust 1.41.1 does not match extension requirement >=1.48.0

      [end of output]

  

  note: This error originates from a subprocess, and is likely not a problem with pip.

  ERROR: Failed building wheel for cryptography

ERROR: Could not build wheels for cryptography, which is required to install pyproject.toml-based projects`
